lms/djangoapps/courseware/views/views.py

```diff
@@ -1060,6 +1060,16 @@ def dates(request, course_id):
         )
         request.user = masquerade_user
 
+    user_access = {
+        'is_enrolled': CourseEnrollment.is_enrolled(request.user, course_key),
+        'is_staff': has_access(request.user, 'staff', course_key),
+    }
```

Comment:
Why put these in a dict like this, rather than just save to two variables?

Reply:
Good question. I was mimicking the way redirecting was handled in the Outline Tab, but I can definitely pare this down.